pavement.py

```diff
@@ -407,7 +407,7 @@ def start_django():
     """
     Start the GeoNode Django application
     """
-    bind = options.get('bind', '')
+    bind = options.get('bind', '0.0.0.0:8000')
```

Comment:
Will be used later by QGIS Server unittests. It needs to access from different network interface.